sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -459,7 +459,7 @@ case class AlterTableSetLocation(
 }
 
 
-private[sql] object DDLUtils {
+object DDLUtils {
```

Comment:
everything in catalyst was meant to be private so we didn't have any private[sql]